packages/backend/server/src/base/mailer/templates/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { render } from '@react-email/render';
```

Comment:
we might put email templates in `src/mails`, the inner path `base/mailer` is not quite straightforward